auth/api/iam/user.go

```diff
@@ -124,10 +149,146 @@ func (r Wrapper) userRedirectStore() storage.SessionStore {
 	return r.storageEngine.GetSessionDatabase().GetStore(userRedirectTimeout, userRedirectSessionKey...)
 }
 
+func (r Wrapper) userSessionStore() storage.SessionStore {
+	return r.storageEngine.GetSessionDatabase().GetStore(userSessionTimeout, userSessionKey...)
+}
+
 func (r Wrapper) oauthClientStateStore() storage.SessionStore {
 	return r.storageEngine.GetSessionDatabase().GetStore(oAuthFlowTimeout, oauthClientStateKey...)
 }
 
 func (r Wrapper) oauthCodeStore() storage.SessionStore {
 	return r.storageEngine.GetSessionDatabase().GetStore(oAuthFlowTimeout, oauthCodeKey...)
 }
+
+// loadUserSession loads the user session given the session ID in the cookie.
+// If there is no session cookie (not yet authenticated, or the session expired), nil is returned.
+// If another, technical error occurs when retrieving the session.
+func (r Wrapper) loadUserSession(ctx echo.Context, tenantDID did.DID) (*UserSession, error) {
```

Comment:
unused.

Reply:
It's used now.

Also used in follow-up PR.